data/json/monsters/insect_spider.json

```diff
@@ -1356,5 +1356,34 @@
     "death_function": [ "NORMAL" ],
     "special_attacks": [ { "type": "leap", "cooldown": 4, "max_range": 4, "allow_no_target": true }, [ "EAT_CROP", 120 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "LARVA", "STUMBLES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_mantis",
+    "type": "MONSTER",
+    "name": { "str": "giant preying mantis" },
+    "description": "A preying mantis, the size of a horse. If you weren't looking closely, you might mistake it for a dead tree.",
+    "default_faction": "mantis",
```

Comment:
I believe you need to declare a `mantis` monster faction but I am not 100% sure.